CHANGES.md

```diff
@@ -6,6 +6,7 @@
 configuration settings should now be accessed via `_converse.api.settings.get` and not directly on the `_converse` object.
 Soon we'll deprecate the latter, so prepare now.
 
+- #2275: Allow selected characters to precede a mention
 - #2231: add sort_by_query and remove sort_by_length
```

Comment:
Please add tickets in the right order in the changelog (I know not all contributors always do that).

Reply:
Yes, no worries. I saw the flow of the document from new to old and instinctively pushed that on the top ^^'